tests/conftest.py

```diff
@@ -148,6 +149,24 @@ def client():
     with TestClient(app) as client:
         yield client
 
+        # Clean up created assets/versions/datasets so teardown doesn't break
```

Comment:
Why does it break?

Reply:
Because during teardown it downgrades all of the migrations during which it attempts to re-instate a unique constraint involving (if I recall correctly) dataset, version, and assettype. Since by the end of some tests we have multiple assets of the same type (in particular raster tile sets), this breaks teardown, leaving a mess. That's why I occasionally mention the possibility of squashing migrations. I initially a implemented something like this at the end of all the RTS/RTC tests, but I decided it was better to clean things out more generally between runs. This also means we don't have dataset collisions between tests if it makes sense to reuse the same names.

Reply:
ok, +1 to squash migrations